HW-09/main.py

```diff
@@ -0,0 +1,45 @@
+# 1. Створити програму, яка буде приймати число і друкувати відповідне число в послідовності Фібоначчі, використовуючи генератори.
+
+number = int(input('Enter a number for function №1: '))
+
+def fibonacci(n):
+    x, y = 0, 1
+    for i in range(n + y):
+        yield x
+        x, y = y, x + y
+
+ls = (list(fibonacci(number)))
+
+print (ls.pop())
```

Comment:
1. Do not use spaces after `print`
2. You can access the last item of a list by slicing, because in most cases you won't need to remove element you are accessing: `ls[-1]`